dashboard/test/ui/features/step_definitions/applab.rb

```diff
@@ -354,3 +366,11 @@ def get_scale_script
 And /^I open the debug console$/ do
   steps 'And I click selector "#debug-area-header .fa-chevron-circle-up" if it exists'
 end
+
+Then(/^the share link includes "([^"]*)"$/) do |expected_text|
+  share_link_input = nil
+  wait_short_until do
+    share_link_input = @browser.find_element(:css, '#sharing-input')
+  end
+  expect(share_link_input.attribute('value')).to include(expected_text)
```

Comment:
Tested that this step fails as expected when the given text is not found:

![image](https://user-images.githubusercontent.com/1615761/33965351-6bc69dd8-e010-11e7-81d9-9f45601b7003.png)